src/YearOverviewHeatmap/index.tsx

```diff
@@ -177,12 +178,22 @@ export function YearOverviewHeatMap({
         .append('path')
         .attr('class', 'month-boundary')
         .attr('fill', 'none')
-        .attr('stroke', '#fff')
+        .attr('stroke', 'var(--background_color)')
         .attr('stroke-width', 3)
         .attr('d', (d) => {
           return getMonthBoundaryPath(d, xScale, yScale);
         });
 
+      // Add text color
+      selectAll('.x-axis , .y-axis')
+        .selectAll('text')
+        .attr('fill', 'var(--primary_color)');
+
+      // Add path color
+      selectAll('.x-axis, .y-axis')
+        .selectAll('path')
+        .attr('stroke', 'var(--primary_color)');
+
```

Comment:
Was the code copied from the private repo or was the change made line-by-line? Copying would be preferred in this case.

Reply:
Yeah sir, I copied the code.